runtime/src/launch/slurm-gasnetrun_ibv/launch-slurm-gasnetrun_ibv.c

```diff
@@ -150,7 +170,10 @@ static void genNumLocalesOptions(FILE* slurmFile, sbatchVersion sbatch,
     if (constraint) {
       fprintf(slurmFile, "#SBATCH --constraint=%s\n", constraint);
     }
-    fprintf(slurmFile, "#SBATCH --exclusive\n");
+    if (oversubscribe == ALLOCATE_SHARED)
+      fprintf(slurmFile, "#SBATCH --oversubscribe\n");
```

Comment:
`--oversubscribe` was only added in slurm 17, and prior to that it was `--share`, so this won't work with older versions of slurm. I'm not really sure what to do about that, but since this something you'd have to opt-in to throw, I'm ok with the incompatibility